diff --git a/node_modules/@notifee/react-native/android/build.gradle b/node_modules/@notifee/react-native/android/build.gradle
index 02a21ae..ba7e4d1 100644
--- a/node_modules/@notifee/react-native/android/build.gradle
+++ b/node_modules/@notifee/react-native/android/build.gradle
@@ -98,10 +98,13 @@ dependencies {
   } else {
     implementation(group: 'app.notifee', name:'core', version: '+')
   }
-  implementation 'androidx.concurrent:concurrent-futures:1.1.0' // https://developer.android.com/jetpack/androidx/releases/concurrent
-  implementation 'androidx.work:work-runtime:2.8.0' // https://developer.android.com/jetpack/androidx/releases/work
+
+  implementation("com.google.guava:guava:33.3.1-android") // https://github.com/google/guava
+
+  implementation 'androidx.concurrent:concurrent-futures:1.2.0' // https://developer.android.com/jetpack/androidx/releases/concurrent
+  implementation 'androidx.work:work-runtime:2.9.1' // https://developer.android.com/jetpack/androidx/releases/work
   implementation 'org.greenrobot:eventbus:3.3.1' // https://github.com/greenrobot/EventBus/releases
-  implementation 'androidx.lifecycle:lifecycle-process:2.3.1'
+  implementation 'androidx.lifecycle:lifecycle-process:2.8.6' // https://developer.android.com/jetpack/androidx/releases/lifecycle
 }
 
 ReactNative.shared.applyPackageVersion()
