diff --git a/node_modules/@react-native-firebase/app/android/build.gradle b/node_modules/@react-native-firebase/app/android/build.gradle
index 8281302..2bcf891 100644
--- a/node_modules/@react-native-firebase/app/android/build.gradle
+++ b/node_modules/@react-native-firebase/app/android/build.gradle
@@ -42,6 +42,11 @@ project.ext {
   ])
 }
 
+
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
   defaultConfig {
     multiDexEnabled true
@@ -71,6 +76,8 @@ dependencies {
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-core"
   implementation "com.google.firebase:firebase-common"
+  def googlePlayServicesAuthVersion = safeExtGet('googlePlayServicesAuthVersion', safeExtGet('googlePlayServicesVersion', '16.0.1'))
+  implementation "com.google.android.gms:play-services-auth:$googlePlayServicesAuthVersion"
 }
 
 apply from: file("./firebase-json.gradle")
